manifest.yaml

```diff
@@ -1,9 +1,11 @@
 id: lnd
 title: LND
-version: 0.17.5.1
+version: 0.18.0
 release-notes: |-
+  * Update to 0.18.0 [Release Notes](https://github.com/lightningnetwork/lnd/releases/tag/v0.18.0-beta)
   * Add Umbrel 1.0 migration action
```

Comment:
shouldn't this be removed?

Reply:
A note about upstream updates is usually included in the service release notes alongside a link to the upstream release notes - why would we not include that note here?

Or are you suggesting the note be something more explicit like `Update LND to 0.18.0`?

Reply:
I think he means "Add Umbrel 1.0 migration action"

Reply:
Correct, this is in reference to the Umbrel action.

Reply:
Ah. That is correct as is.

The Umbrel 1.0 migration was merged into main in a previous [PR](https://github.com/Start9Labs/lnd-startos/pull/124), but 0.18.0 will be the first release including this action.